### ISSUE TITLE
View Change Failed in mainnet

### ISSUE THREAD

**Issue by rlan35** [harmony-one/harmony#3148]
Seen lots of this msgs in all nodes:

```{"level":"info","port":"9000","ip":"3.112.219.248","myBlock":3597371,"myViewID":3597493,"phase":2,"mode":"ViewChanging","caller":"/home/leochen/go/src/github.com/harmony-one/harmony/consensus/checks.go:200","time":"2020-06-10T00:53:03.555523452Z","message":"[VIEWCHANGE] sender key not in this slot's subcommittee"}```

and also this msg:
```wrap
{"level":"info","port":"9000","ip":"35.189.91.148","myBlock":3597371,"myViewID":3597493,"phase":2,"mode":"Normal","caller":"/home/leochen/go/src/github.com/harmony-one/harmony/consensus/view_change.go:627","time":"2020-06-10T00:50:52.553409098Z","message":"onNewView === commit"}
{"level":"warn","port":"9000","ip":"35.189.91.148","myBlock":3597371,"myViewID":3597493,"phase":2,"mode":"Normal","LastSuccessfulConsensusViewID":3597493,"MsgViewChangingID":3597493,"caller":"/home/leochen/go/src/github.com/harmony-one/harmony/consensus/checks.go:260","time":"2020-06-10T00:51:01.885218663Z","message":"[onNewView] ViewID should be larger than the viewID of the last successful consensus"}```

**Comment by rlan35** [comment]
`{"level":"warn","port":"9000","ip":"52.212.32.116","myBlock":3597371,"myViewID":3597493,"phase":2,"mode":"Normal","LastSuccessfulConsensusViewID":3597493,"MsgViewChangingID":3597493,"caller":"/home/leochen/go/src/github.com/harmony-one/harmony/consensus/checks.go:260","time":"2020-06-10T00:51:51.868004966Z","message":"[onNewView] ViewID should be larger than the viewID of the last successful consensus"}
--- END 52.212.32.116 stdout ---
--- BEGIN 18.196.17.123 stderr ---
ssh: connect to host 18.196.17.123 port 22: Connection timed out
--- END 18.196.17.123 stderr ---
18.196.17.123 returned status 255
--- BEGIN 3.126.254.238 stderr ---
ssh: connect to host 3.126.254.238 port 22: Connection timed out
--- END 3.126.254.238 stderr ---
3.126.254.238 returned status 255
--- BEGIN 18.202.228.50 stdout ---
{"level":"warn","port":"9000","ip":"18.202.228.50","myBlock":3597371,"myViewID":3597493,"phase":2,"mode":"Normal","LastSuccessfulConsensusViewID":3597493,"MsgViewChangingID":3597493,"caller":"/home/leochen/go/src/github.com/harmony-one/harmony/consensus/checks.go:260","time":"2020-06-10T00:51:51.917967314Z","message":"[onNewView] ViewID should be larger than the viewID of the last successful consensus"}
--- END 18.202.228.50 stdout ---
--- BEGIN 52.212.0.190 stdout ---
{"level":"warn","port":"9000","ip":"52.212.0.190","myBlock":3597371,"myViewID":3597493,"phase":2,"mode":"Normal","LastSuccessfulConsensusViewID":3597493,"MsgViewChangingID":3597493,"caller":"/home/leochen/go/src/github.com/harmony-one/harmony/consensus/checks.go:260","time":"2020-06-10T00:51:51.871544928Z","message":"[onNewView] ViewID should be larger than the viewID of the last successful consensus"}
--- END 52.212.0.190 stdout ---
--- BEGIN 34.218.48.95 stdout ---
{"level":"warn","port":"9000","ip":"34.218.48.95","myBlock":3597371,"myViewID":3597493,"phase":2,"mode":"Normal","LastSuccessfulConsensusViewID":3597493,"MsgViewChangingID":3597493,"caller":"/home/leochen/go/src/github.com/harmony-one/harmony/consensus/checks.go:260","time":"2020-06-10T00:51:51.941884844Z","message":"[onNewView] ViewID should be larger than the viewID of the last successful consensus"}
--- END 34.218.48.95 stdout ---
--- BEGIN 3.120.152.149 stderr ---
ssh: connect to host 3.120.152.149 port 22: Connection timed out
--- END 3.120.152.149 stderr ---
3.120.152.149 returned status 255
--- BEGIN 3.125.160.197 stderr ---
ssh: connect to host 3.125.160.197 port 22: Connection timed out
--- END 3.125.160.197 stderr ---
3.125.160.197 returned status 255
--- BEGIN 34.251.240.119 stdout ---
{"level":"warn","port":"9000","ip":"34.251.240.119","myBlock":3597371,"myViewID":3597493,"phase":2,"mode":"Normal","LastSuccessfulConsensusViewID":3597493,"MsgViewChangingID":3597493,"caller":"/home/leochen/go/src/github.com/harmony-one/harmony/consensus/checks.go:260","time":"2020-06-10T00:51:51.873528905Z","message":"[onNewView] ViewID should be larger than the viewID of the last successful consensus"}
--- END 34.251.240.119 stdout ---
--- BEGIN 34.253.235.135 stdout ---
{"level":"warn","port":"9000","ip":"34.253.235.135","myBlock":3597371,"myViewID":3597493,"phase":2,"mode":"Normal","LastSuccessfulConsensusViewID":3597493,"MsgViewChangingID":3597493,"caller":"/home/leochen/go/src/github.com/harmony-one/harmony/consensus/checks.go:260","time":"2020-06-10T00:51:51.873967987Z","message":"[onNewView] ViewID should be larger than the viewID of the last successful consensus"}
--- END 34.253.235.135 stdout ---
--- BEGIN 3.122.177.187 stderr ---
@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
@    WARNING: REMOTE HOST IDENTIFICATION HAS CHANGED!     @
@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@
IT IS POSSIBLE THAT SOMEONE IS DOING SOMETHING NASTY!
Someone could be eavesdropping on you right now (man-in-the-middle attack)!
It is also possible that a host key has just been changed.
The fingerprint for the ECDSA key sent by the remote host is
SHA256:eRkLW0uov5/kMbEoUTcKxwonU9sfaXEcKYw8EYG7KXg.
Please contact your system administrator.
Add correct host key in logs/s3/known_hosts to get rid of this message.
Offending ECDSA key in logs/s3/known_hosts:59
Password authentication is disabled to avoid man-in-the-middle attacks.
Keyboard-interactive authentication is disabled to avoid man-in-the-middle attacks.
Permission denied (publickey,gssapi-keyex,gssapi-with-mic).
--- END 3.122.177.187 stderr ---
3.122.177.187 returned status 255
--- BEGIN 34.244.203.58 stdout ---
{"level":"warn","port":"9000","ip":"34.244.203.58","myBlock":3597371,"myViewID":3597493,"phase":2,"mode":"Normal","LastSuccessfulConsensusViewID":3597493,"MsgViewChangingID":3597493,"caller":"/home/leochen/go/src/github.com/harmony-one/harmony/consensus/checks.go:260","time":"2020-06-10T00:51:51.8943537Z","message":"[onNewView] ViewID should be larger than the viewID of the last successful consensus"}
--- END 34.244.203.58 stdout ---
--- BEGIN 63.35.225.140 stdout ---
{"level":"warn","port":"9000","ip":"63.35.225.140","myBlock":3597371,"myViewID":3597493,"phase":2,"mode":"Normal","LastSuccessfulConsensusViewID":3597493,"MsgViewChangingID":3597493,"caller":"/home/leochen/go/src/github.com/harmony-one/harmony/consensus/checks.go:260","time":"2020-06-10T00:51:51.879681573Z","message":"[onNewView] ViewID should be larger than the viewID of the last successful consensus"}
--- END 63.35.225.140 stdout ---
--- BEGIN 34.221.104.186 stdout ---
{"level":"warn","port":"9000","ip":"34.221.104.186","myBlock":3597371,"myViewID":3597493,"phase":2,"mode":"Normal","LastSuccessfulConsensusViewID":3597493,"MsgViewChangingID":3597493,"caller":"/home/leochen/go/src/github.com/harmony-one/harmony/consensus/checks.go:260","time":"2020-06-10T00:51:51.866335222Z","message":"[onNewView] ViewID should be larger than the viewID of the last successful consensus"}
--- END 34.221.104.186 stdout ---
--- BEGIN 18.202.231.246 stdout ---
{"level":"warn","port":"9000","ip":"18.202.231.246","myBlock":3597371,"myViewID":3597493,"phase":2,"mode":"Normal","LastSuccessfulConsensusViewID":3597493,"MsgViewChangingID":3597493,"caller":"/home/leochen/go/src/github.com/harmony-one/harmony/consensus/checks.go:260","time":"2020-06-10T00:51:51.868100043Z","message":"[onNewView] ViewID should be larger than the viewID of the last successful consensus"}
--- END 18.202.231.246 stdout ---
--- BEGIN 18.194.37.153 stderr ---
ssh: connect to host 18.194.37.153 port 22: Connection timed out
--- END 18.194.37.153 stderr ---
18.194.37.153 returned status 255
--- BEGIN 54.212.72.201 stderr ---
ssh: connect to host 54.212.72.201 port 22: Connection timed out
--- END 54.212.72.201 stderr ---
54.212.72.201 returned status 255
--- BEGIN 3.0.19.33 stdout ---
{"level":"warn","port":"9000","ip":"3.0.19.33","myBlock":3597371,"myViewID":3597493,"phase":2,"mode":"Normal","LastSuccessfulConsensusViewID":3597493,"MsgViewChangingID":3597493,"caller":"/home/leochen/go/src/github.com/harmony-one/harmony/consensus/checks.go:260","time":"2020-06-10T00:51:51.946589365Z","message":"[onNewView] ViewID should be larger than the viewID of the last successful consensus"}
--- END 3.0.19.33 stdout ---
--- BEGIN 13.229.205.39 stdout ---
{"level":"warn","port":"9000","ip":"13.229.205.39","myBlock":3597371,"myViewID":3597493,"phase":2,"mode":"Normal","LastSuccessfulConsensusViewID":3597493,"MsgViewChangingID":3597493,"caller":"/home/leochen/go/src/github.com/harmony-one/harmony/consensus/checks.go:260","time":"2020-06-10T00:51:51.968880933Z","message":"[onNewView] ViewID should be larger than the viewID of the last successful consensus"}
--- END 13.229.205.39 stdout ---
--- BEGIN 52.37.59.120 stderr ---
ssh: connect to host 52.37.59.120 port 22: Connection timed out
--- END 52.37.59.120 stderr ---
52.37.59.120 returned status 255
--- BEGIN 52.91.158.242 stderr ---
ssh: connect to host 52.91.158.242 port 22: Connection timed out
--- END 52.91.158.242 stderr ---
52.91.158.242 returned status 255
--- BEGIN 54.213.202.239 stderr ---
ssh: connect to host 54.213.202.239 port 22: Connection timed out
--- END 54.213.202.239 stderr ---
54.213.202.239 returned status 255
--- BEGIN 54.95.38.16 stderr ---
ssh: connect to host 54.95.38.16 port 22: Connection timed out
--- END 54.95.38.16 stderr ---
54.95.38.16 returned status 255
--- BEGIN 104.199.209.190 stdout ---
{"level":"warn","port":"9000","ip":"104.199.209.190","myBlock":3597371,"myViewID":3597493,"phase":2,"mode":"Normal","LastSuccessfulConsensusViewID":3597493,"MsgViewChangingID":3597493,"caller":"/home/leochen/go/src/github.com/harmony-one/harmony/consensus/checks.go:260","time":"2020-06-10T00:51:51.999248101Z","message":"[onNewView] ViewID should be larger than the viewID of the last successful consensus"}
--- END 104.199.209.190 stdout ---
--- BEGIN 35.189.91.148 stdout ---
{"level":"warn","port":"9000","ip":"35.189.91.148","myBlock":3597371,"myViewID":3597493,"phase":2,"mode":"Normal","LastSuccessfulConsensusViewID":3597493,"MsgViewChangingID":3597493,"caller":"/home/leochen/go/src/github.com/harmony-one/harmony/consensus/checks.go:260","time":"2020-06-10T00:51:51.885699689Z","message":"[onNewView] ViewID should be larger than the viewID of the last successful consensus"}
--- END 35.189.91.148 stdout ---
--- BEGIN 35.198.236.247 stdout ---
{"level":"warn","port":"9000","ip":"35.198.236.247","myBlock":3597371,"myViewID":3597493,"phase":2,"mode":"Normal","LastSuccessfulConsensusViewID":3597493,"MsgViewChangingID":3597493,"caller":"/home/leochen/go/src/github.com/harmony-one/harmony/consensus/checks.go:260","time":"2020-06-10T00:51:51.99447583Z","message":"[onNewView] ViewID should be larger than the viewID of the last successful consensus"}
--- END 35.198.236.247 stdout ---
--- BEGIN 35.204.5.5 stdout ---
{"level":"warn","port":"9000","ip":"35.204.5.5","myBlock":3597371,"myViewID":3597493,"phase":2,"mode":"Normal","LastSuccessfulConsensusViewID":3597493,"MsgViewChangingID":3597493,"caller":"/home/leochen/go/src/github.com/harmony-one/harmony/consensus/checks.go:260","time":"2020-06-10T00:51:51.909846218Z","message":"[onNewView] ViewID should be larger than the viewID of the last successful consensus"}
--- END 35.204.5.5 stdout ---
--- BEGIN 35.229.213.122 stdout ---
{"level":"warn","port":"9000","ip":"35.229.213.122","myBlock":3597371,"myViewID":3597493,"phase":2,"mode":"Normal","LastSuccessfulConsensusViewID":3597493,"MsgViewChangingID":3597493,"caller":"/home/leochen/go/src/github.com/harmony-one/harmony/consensus/checks.go:260","time":"2020-06-10T00:51:51.943246988Z","message":"[onNewView] ViewID should be larger than the viewID of the last successful consensus"}
--- END 35.229.213.122 stdout ---
--- BEGIN 35.246.20.148 stdout ---
{"level":"warn","port":"9000","ip":"35.246.20.148","myBlock":3597371,"myViewID":3597493,"phase":2,"mode":"Normal","LastSuccessfulConsensusViewID":3597493,"MsgViewChangingID":3597493,"caller":"/home/leochen/go/src/github.com/harmony-one/harmony/consensus/checks.go:260","time":"2020-06-10T00:51:51.875463836Z","message":"[onNewView] ViewID should be larger than the viewID of the last successful consensus"}`

basically all nodes failed with the viewID check. Likely the new leader didn't set the current ViewID.

**Comment by LeoHChen** [comment]
https://github.com/harmony-one/harmony/issues/3149
